packages/block-library/src/cover/edit.native.js

```diff
@@ -252,9 +252,12 @@ const Cover = ( {
 		</BlockControls>
 	);
 
-	const addMediaButton = ( open ) => (
+	const addMediaButton = () => (
 		<View style={ styles.selectImageContainer }>
```

Comment:
We could remove the double `styles.selectImageContainer` by wrapping the view like this:

```patch
diff --git a/packages/block-library/src/cover/edit.native.js b/packages/block-library/src/cover/edit.native.js
index 14b413712a..d698791df7 100644
--- a/packages/block-library/src/cover/edit.native.js
+++ b/packages/block-library/src/cover/edit.native.js
@@ -253,11 +253,8 @@ const Cover = ( {
 	);
 
 	const addMediaButton = () => (
-		<View style={ styles.selectImageContainer }>
-			<TouchableWithoutFeedback
-				style={ styles.selectImageContainer }
-				onPress={ openMediaOptionsRef.current }
-			>
+		<TouchableWithoutFeedback onPress={ openMediaOptionsRef.current }>
+			<View style={ styles.selectImageContainer }>
 				<View style={ styles.selectImage }>
 					<Icon
 						size={ 16 }
@@ -265,8 +262,8 @@ const Cover = ( {
 						{ ...styles.selectImageIcon }
 					/>
 				</View>
-			</TouchableWithoutFeedback>
-		</View>
+			</View>
+		</TouchableWithoutFeedback>
 	);
 
 	const controls = (

```

Reply:
Thank you for the suggestion @mkevins! This looks much better. I updated the PR